src/core/main.go

```diff
@@ -228,11 +243,12 @@ func main() {
 		if err := scan.EnsureScanner(reg, true); err != nil {
 			log.Fatalf("failed to initialize clair scanner: %v", err)
 		}
-	} else {
```

Comment:
@steven-zou @heww  I wasn't sure why we're deleting all immutable scanners, when `config.WithClair()` returns `false`. Temporarily commented this out so the Trivy scanner is not removed by this condition.

Reply:
This need changed to remove clair scanner when `config.WithClair` returns false, and remove trivy scanner when `config.WithTrivy` returns false.

Reply:
Done.

Reply:
> This need changed to remove clair scanner when `config.WithClair` returns false, and remove trivy scanner when `config.WithTrivy` returns false.

Done